source/includes/options-mongod.yaml

```diff
@@ -1140,6 +1140,20 @@ description: |
   Configures this {{program}} instance as a shard in a
   sharded cluster. The default port for these instances is
   ``27018``.
+
+  .. important::
+
+     .. versionchanged:: 3.6
+
+     You must deploy shard servers as a replica set with the
+     :option:`--replSet` option specified. To deploy sharded clusters
+     with replication, follow the appropriate tutorial depending on your
+     :ref:`sharding strategy<sharding-strategy>`:
+
+     - :doc:`/tutorial/deploy-sharded-cluster-hashed-sharding`
+
+     - :doc:`/tutorial/deploy-sharded-cluster-ranged-sharding`
```

Comment:
We should also delete  the standalone blurb on the tutorials (see end of section)

https://docs-mongodbcom-staging.corp.mongodb.com/jdestefano/DOCS-11064/tutorial/deploy-sharded-cluster-hashed-sharding.html#add-shards-to-the-cluster

https://docs-mongodbcom-staging.corp.mongodb.com/jdestefano/DOCS-11064/tutorial/deploy-sharded-cluster-ranged-sharding.html#add-shards-to-the-cluster

Also, both
https://docs-mongodbcom-staging.corp.mongodb.com/jdestefano/DOCS-11064/core/sharded-cluster-components.html

https://docs-mongodbcom-staging.corp.mongodb.com/jdestefano/DOCS-11064/core/sharded-cluster-shards.html


 Each shard can be deployed as a replica set.   (no longer **can** be -> must be)